src/pages/TechnicalSettings/TechnicalSettings.tsx

```diff
@@ -22,6 +22,36 @@ const TechnicalSettings: FC = () => {
   const { institutionDiscounts } = useFetchInstitutionDiscounts()
   const { updateInstitutionDiscounts } = useUpdateInstitutionDiscounts()
 
+  const defaultValues = useMemo(
```

Comment:
**[non-blocker]**

Quite a lot of repetition here, would suggest using something to map these from institutionDiscounts, to avoid repetition. For example with lodash mapValues it would be:

const defaultValues = mapValues(institutionDiscounts, (value) => String(value) || '0')

Reply:
Ooh fancy, will use that 👍